### DISCUSSION TITLE
Science behind "Stability"

### DISCUSSION THREAD

**Discussion by svscorp** [nbrown02/FlowViz-Jira#36] [Q&A] [answered]
I am curious, what is used behind the "Stability" in forecast, and the advice to keep it below 25%?

**Comment by nbrown02** [comment, marked as answer]
I took the same logic from Troy Magennis' work:

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/1368a1fe-dfc9-4a28-87de-29d99540d097)

**Reply by svscorp** [reply]
So stability, it's the "error" (variation/deviation) then? Usually high stability is good, if the aim for this stability to keep it lower, then considering it as deviation/variation makes sense. Or I might still misunderstand the point.

**Reply by nbrown02** [reply]
Yup - I could/should probably change it actually to be something clearer (reverse the calculation and do "aim for 75%+") for example
